Gruntfile.js

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2017 OCAD University
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENSE.txt
+*/
+
+// Declare dependencies
+/* global module */
+
+module.exports = function (grunt) {
+    "use strict";
+
+    // Project configuration.
+    grunt.initConfig({
+        // Project package file destination.
+        pkg: grunt.file.readJSON("package.json"),
+        eslint: {
+            all: ["src/**/*.js", "tests/**/*.js", "demos/**/*.js", "examples/**/*.js", "*.js"]
+        },
+        jsonlint: {
+            all: ["package.json", ".jshintrc", "src/**/*.json", "tests/**/*.json", "demos/**/*.json", "!node_modules", "!src/lib/**", "!tests/lib/**"]
```

Comment:
`".jshintrc"` should be `".eslintrc.json"`.

And we can remove these entries: `"demos/**/*.json"`, `"!node_modules"`, `"!src/lib/**"`, and `"!tests/lib/**"`.